opm/simulators/flow/BlackoilModelConvergenceMonitor.hpp

```diff
@@ -0,0 +1,53 @@
+/*
+  Copyright 2013, 2015 SINTEF ICT, Applied Mathematics.
```

Comment:
I think this list of copyright holders is incorrect. It is a new facility, and does not inherit the old copyrights.

Reply:
I am sorry but I cannot change the copyright on code, I can only repeat what was there. The file is new, the contents are not.